javascript/javascript1/week3/smartphoneusage.js

```diff
@@ -0,0 +1,61 @@
+const activities = [];
+const usageLimit = 120;
+function addActivity(activity, duration) {
+  activities.push({
+    date: new Date().toLocaleDateString("en-US"),
+    activity: activity,
+    duration: duration,
+  });
+}
+
+function showStatus(activities) {
```

Comment:
This function looks so untidy,  I think if you split operations, makes it easier to read.